pulumi/grapl/__main__.py

```diff
@@ -310,6 +311,10 @@ def main() -> None:
         ###################################
         # Local Grapl
         ###################################
+        organization_management_db = LocalPostgresInstance(
+            name="organization_management_db",
+            port=5632,
```

Comment:
Is there a reason this is defined both here and in `local-grapl.env`? It would be better to have a single source of truth.

Reply:
Not sure, @colin-grapl  - any input?^^

Reply:
I have no idea how all of these things work together. I don't know how you would access the env here.